main.cpp

```diff
@@ -120,6 +122,22 @@ OvenStove::FoodItem::FoodItem() : type("Poultry"),
     std::cout << "----------constructing FoodItem----------" << "\n";
 }
 
+void OvenStove::FoodItem::storeInFridge(int threshold, int expirationDuration){
+    expirationDayRemain = expirationDuration;
+    
+    while (expirationDayRemain > threshold)
+    {
+        std::cout << "There are " << expirationDayRemain << " days till expired. Keep in fridge.\n";
+        expirationDayRemain--;
```

Comment:
- Curly braces go on their own line with no other code
- use `--i`, not `i--`
- [ ] revise